Misc/NEWS.d/next/Library/2020-02-28-12-59-30.bpo-39678.3idfxM.rst

```diff
@@ -0,0 +1 @@
+Refactor queue_manager in concurrent.futures.process to make it easier to maintain.
```

Comment:
Most users are likely going to be more familiar with ProcessPoolExecutor rather than the internal file name that it happens to be contained within. Also, by using the actual name, we can convert it into an inline link to it's documentation w/ Sphinx markup. 

```suggestion
Refactor queue_manager in :class:`concurrent.futures.ProcessPoolExecutor` to
make it easier to maintain.
```

(You can also optionally include a `Contributed by <name>.` at the end of the Misc/NEWS entry.)